README

```diff
@@ -1,5 +1,11 @@
-UCX is a communication library implementing a high-performance messaging
-layer for MPI, PGAS, and RPC frameworks.
+UCX stands for Unified Communication X. UCX provides an optimized communication
+layer for Message Passing (MPI), PGAS/OpenSHMEM libraries and RPC/data-centric
+applications. UCX utilizes high-speed networks, such as RDMA (InfiniBand, RoCE,
+etc), Cray Gemini or Aries, for inter-node communication. If no such network is
+available, TCP is used instead. UCX supports efficient transfer of data in
+either main memory (RAM) or GPU memory (through CUDA and RoCM libraries).
```

Comment:
s/RoCM/ROCm/

Reply:
@yosefe Ok. I guess has to be ported to master as well